server/bin/pbench-dispatch.sh

```diff
@@ -185,6 +189,22 @@ while read tbmd5; do
         continue
     fi
 
+    if [[ ! -z "${put_token}" ]]; then
+        # We have a bearer token for the PUT API of a "new" Pbench Server,
+        # invoke the `pbench-results-push` agent CLI to send it to the
+        # server (configuration of that server handled elsewhere).
+        satellite=${controller%%::*}
+        if [[ "${controller}" != "${satellite}" ]]; then
+            metadata_arg="--metadata=server.origin:${satellite}"
+        fi
+        pbench-results-push ${tb} --token ${put_token} ${metadata_arg}
```

Comment:
should we also add `--access=public` in here? if we always wants to send tarball with public access from `0.69 server` to `1.0 server`

Reply:
Yes. And while we've been using `server.origin` for satellite, at one point it had occurred to me that it might be valuable to also tag the put-shim server, maybe using a `<satellite>:<put-shim>` syntax. (Or '[\<satellite>::]\<put-shim>`?)

Reply:
Added, see what you think.